// @qumra/web-pixels-extension/dist/types/PixelEvents.js
// ============================================================================
// Enums
// ============================================================================
export var EventType;
(function (EventType) {
    EventType["Standard"] = "standard";
    EventType["Custom"] = "custom";
    EventType["Dom"] = "dom";
})(EventType || (EventType = {}));
export var AlertDisplayedType;
(function (AlertDisplayedType) {
    AlertDisplayedType["CheckoutError"] = "CHECKOUT_ERROR";
    AlertDisplayedType["ContactError"] = "CONTACT_ERROR";
    AlertDisplayedType["DeliveryError"] = "DELIVERY_ERROR";
    AlertDisplayedType["DiscountError"] = "DISCOUNT_ERROR";
    AlertDisplayedType["InputInvalid"] = "INPUT_INVALID";
    AlertDisplayedType["InputRequired"] = "INPUT_REQUIRED";
    AlertDisplayedType["InventoryError"] = "INVENTORY_ERROR";
    AlertDisplayedType["MerchandiseError"] = "MERCHANDISE_ERROR";
    AlertDisplayedType["PaymentError"] = "PAYMENT_ERROR";
})(AlertDisplayedType || (AlertDisplayedType = {}));
